extensions/wikia/Captcha/Captcha.setup.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+/**
+ * @file
+ * @ingroup Extensions
+ */
+
+if ( !defined( 'MEDIAWIKI' ) ) {
+	exit;
+}
+
+$wgExtensionCredits['captcha'][] = [
+	'path' => __FILE__,
+	'name' => 'Captcha',
+	'author' => ['Garth Webb'],
+	'descriptionmsg' => 'captcha-desc',
+];
+
+/**
+ * The 'skipcaptcha' permission key can be given out to
+ * let known-good users perform triggering actions without
+ * having to go through the captcha.
+ *
+ * By default, sysops and registered bot accounts will be
+ * able to skip, while others have to go through it.
+ */
+$wgGroupPermissions['*'            ]['skipcaptcha'] = false;
+$wgGroupPermissions['user'         ]['skipcaptcha'] = false;
```

Comment:
Its copied code, so I kept it because it was already there and does actually make it easier to read.  I'd prefer it this way if the IDE did it for us but its a pain to keep stuff like this around so I'll delete them for consistency.

As far as the settings, I think they're all here to enumerate the options.  I'll get rid of the `user` and `autoconfirmed` lines.